src/content/en/updates/2018/05/webauthn.md

```diff
@@ -0,0 +1,164 @@
+response to this problem has been multi-factor authentication, but 
+implementations are fragmented and most still don't adequately address phishing. 
+We have been working with the [FIDO Alliance](https://fidoalliance.org/) since 
+2013 and, more recently, with the W3C to implement a standardized 
+phishing-resistant protocol that can be used by any Web application.
+
+## What is WebAuthn?
+
+[The Web Authentication API](https://www.w3.org/TR/webauthn/) gives Web 
+applications user-agent-mediated access to authenticators – which are often 
+hardware tokens accessed over USB/BLE/NFC or modules built directly into the 
+platform – for the purposes of generating and challenging application-scoped 
+(eTLD+k) public-key credentials. This enables a variety of use-cases, such as:
+
+* Low friction and phishing-resistant 2FA (to be used in conjunction with 
+  password)
+* Passwordless, biometrics-based re-authorization
+* Low friction and phishing-resistant 2FA **without** password (to be used for 
+  passwordless accounts)
+
+The API is on track to be implemented by most major browsers, and is intended to 
+both simplify the UI encountered when having to prove your identity online and 
+significantly reduce phishing.
+
+WebAuthn extends the [Credential Management 
+API](https://w3c.github.io/webappsec-credential-management/) and adds a new 
+credential type called `PublicKeyCredential`. WebAuthn abstracts the 
+communication between the browser and an authenticator and allows a user to:
+
+1. Create and register a public key credential for a website
+1. Authenticate to a website by proving possession of the corresponding private 
+   key
+
+Authenticators are devices that can generate private/public key pairs and gather 
+consent. Consent for signing can be granted with a simple tap, a successful 
+fingerprint read, or by other methods as long as they comply with FIDO2 
+requirements (there's a [certification 
+program](https://fidoalliance.org/certification/fido-certified-products/) for 
+authenticators by the FIDO Alliance). Authenticators can either be built into 
+the platform (such as fingerprint scanners on smartphones) or attached through 
+USB, Bluetooth Low Energy (BLE), or Near-Field Communication (NFC). 
+
+## How it works
+
+### Creating a key pair and registering a user
+
+When a user wants to register a credential to a website (referred to by WebAuthn 
+as the "relying party"):
+
+1. The relying party generates a challenge.
+1. The relying party asks the browser, through the Credential Manager API, to 
+   generate a new credential for the relying party, specifying device 
+   capabilities, e.g., whether the device provides its own user authentication 
+   (with biometrics, etc).
+1. After the authenticator obtains user consent, the authenticator generates a
+   key pair and returns the public key, optional signed attestation, and
+   transport information to the website.
```

Comment:
agl: "registration doesn't currently include transport information. We're working on adding it so maybe we're documenting the world we want to see"

Reply:
We are indeed :) 

Eiji, maybe change to:
"key pair and returns the public key and optional signed attestation to the website."